### ISSUE TITLE
Add ability to put Hyperion into "expected beam off mode"

### ISSUE THREAD

**Issue by DominicOram** [DiamondLightSource/hyperion#1062]
When testing on a Tuesday there will be a number of things that we know will not be working but we would like Hyperion to carry on regardless. Examples of these so far are:
* XBPM feedback not locking (we get round this by writing to the feedback threshold PV but we could guard against it in Hyperion)
* Undulator gap being disabled

We want to be very certain that we're not in this mode during normal operation. The best way to do this, (I think but obviously open to other suggestions) is to have an endpoint to Hyperion where it will only put it in this mode for a fixed length of time and then will switch back afterwards

## Acceptance Criteria
* There is a way to tell Hyperion to go into a mode where it ignores these things for a certain length of time
* The above are factored in on this mode

**Comment by DominicOram** [comment]
For the implementation I wonder if we can do it through creating some ophyd devices in this mode

**Comment by rtuck99** [comment]
I agree with the timeout but we should still have a way of manually disabling test mode and the timeout should just be a fallback in case someone forgets.
